faucetagent.py

```diff
@@ -330,12 +330,13 @@ def parse():
     arg = parser.add_argument
     arg('--cert', required=True, help='certificate file')
     arg('--key', required=True, help='private key file')
-    arg('--gnmiport', type=int, default=10161, help='gNMI port (10161)')
+    arg('--gnmi_addr', default='[::]', help='gNMI address to listen on ([::])')
+    arg('--gnmi_port', type=int, default=10161, help='gNMI port (10161)')
```

Comment:
I prefer `--gnmiport` as it's easier to type. I realize that internally we are following the irritating PEP8 convention of using underscores, but that seems to be the FAUCET convention and is also what pylint wants.